pkg/api/v1beta1/conversion.go

```diff
@@ -731,6 +731,21 @@ func init() {
 			}
 			return nil
 		},
+		func(in *Namespace, out *newer.Namespace, s conversion.Scope) error {
```

Comment:
This is different from other funcs above.  Is it correct, and if so does it deserve a comment?